src/ORM/ResetDatabase/BaseOrmResetter.php

```diff
@@ -30,6 +35,19 @@ public function __construct(
     ) {
     }
 
+    final public function resetBeforeEachTest(KernelInterface $kernel): void
+    {
+        if (self::$inFirstTest) {
+            self::$inFirstTest = false;
+
+            return;
+        }
```

Comment:
prevents the to run twice the schema create in the first test, when not using dama.